PE/tests/test_version.py

```diff
@@ -29,3 +29,29 @@ def test_pe_vsersion(self):
         self.assertEqual(len(keys), len(vs_version))
         for key in vs.getVersionKeys():
             self.assertEqual(vs_version.get(key), vs.getVersionValue(key))
+
+    def test_export_by_name(self):
+        fpath = os.path.join('bins','export_by_name.dll')
```

Comment:
These are causing the unit tests to fail ( probably because the dir names should be PE/tests/bin/ )